.mvn/maven.config

```diff
@@ -1,3 +1,4 @@
 --batch-mode
 --errors
 --strict-checksums
+-Dstyle.color=always
```

Comment:
Ah yes wasn't meant to sneak that in, but more to start the discussion. GHA has nice support; if not applied through `.mvn/config` it can instead be added to the `.github/workflows` for the same effect in GHA.

Reply:
Indeed, that's the kind of trade-off we're debating internally :+1:

(Alternatively we may omit `--batch-mode` from this file, then add it e.g. for CI.)